utils/config.ts

```diff
@@ -83,6 +83,9 @@ export const FeatureFlagFromString = withFallback(
 export type IConfig = t.TypeOf<typeof IConfig>;
 export const IConfig = t.intersection([
   t.interface({
+    AZURE_STORAGE_ACCOUNT_KEY: NonEmptyString,
```

Comment:
this name is a little bit too generic. if this storage is used only for email check use a more specific name, something like `PROFILE_EMAIL_STORAGE_ACCOUNT_KEY`